docs/id-lifecycle-management/support-systems/partner-management-services/functional-overview/onboard-credential-partner.md

```
@@ -176,25 +176,25 @@ Partner admin should create the biometric extraction provider configuration so t
 
 **Step 3: Approve Policy request Submitted by Credential Partner**
 
-7. Once the policy request is submitted by the credential partner, partner admin can view the list of pending request by clicking on "Partner-Policy Linking" card.
+1. Once the policy request is submitted by the credential partner, partner admin can view the list of pending request by clicking on "Partner-Policy Linking" card.
 
 ![](../../../../.gitbook/assets/pms-b5-ocp-image23.png)
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)